frame/staking/src/default_weights.rs

```diff
@@ -0,0 +1,249 @@
+			.saturating_add((6474000 as Weight).saturating_mul(w as Weight))
+			.saturating_add(DbWeight::get().reads(6 as Weight))
+			.saturating_add(DbWeight::get().reads((4 as Weight).saturating_mul(a as Weight)))
+			.saturating_add(DbWeight::get().reads((1 as Weight).saturating_mul(w as Weight)))
+			.saturating_add(DbWeight::get().writes(2 as Weight))
+	}
+	fn submit_solution_better(v: u32, n: u32, a: u32, w: u32, ) -> Weight {
+		(0 as Weight)
+			.saturating_add((784000 as Weight).saturating_mul(v as Weight))
+			.saturating_add((640000 as Weight).saturating_mul(n as Weight))
+			.saturating_add((68830000 as Weight).saturating_mul(a as Weight))
+			.saturating_add((8181000 as Weight).saturating_mul(w as Weight))
+			.saturating_add(DbWeight::get().reads(6 as Weight))
+			.saturating_add(DbWeight::get().reads((4 as Weight).saturating_mul(a as Weight)))
+			.saturating_add(DbWeight::get().reads((1 as Weight).saturating_mul(w as Weight)))
+			.saturating_add(DbWeight::get().writes(2 as Weight))
+	}
+	fn submit_solution_weaker(v: u32, n: u32, ) -> Weight {
+		(11244000 as Weight)
+			.saturating_add((28000 as Weight).saturating_mul(v as Weight))
+			.saturating_add((12000 as Weight).saturating_mul(n as Weight))
+			.saturating_add(DbWeight::get().reads(3 as Weight))
+	}
+}
+
+
+#[cfg(feature = "std")]
+impl crate::WeightInfo for () {
+	fn bond() -> Weight { 1_000_000_000 }
+	fn bond_extra() -> Weight { 1_000_000_000 }
+	fn unbond() -> Weight { 1_000_000_000 }
+	fn withdraw_unbonded_update(_s: u32, ) -> Weight { 1_000_000_000 }
+	fn withdraw_unbonded_kill(_s: u32, ) -> Weight { 1_000_000_000 }
+	fn validate() -> Weight { 1_000_000_000 }
+	fn nominate(_n: u32) -> Weight { 1_000_000_000 }
+	fn chill() -> Weight { 1_000_000_000 }
+	fn set_payee() -> Weight { 1_000_000_000 }
+	fn set_controller() -> Weight { 1_000_000_000 }
+	fn set_validator_count() -> Weight { 1_000_000_000 }
+	fn force_no_eras() -> Weight { 1_000_000_000 }
+	fn force_new_era() -> Weight { 1_000_000_000 }
+	fn force_new_era_always() -> Weight { 1_000_000_000 }
+	fn set_invulnerables(_v: u32, ) -> Weight { 1_000_000_000 }
+	fn force_unstake(_s: u32, ) -> Weight { 1_000_000_000 }
+	fn cancel_deferred_slash(_s: u32, ) -> Weight { 1_000_000_000 }
+	fn payout_stakers(_n: u32, ) -> Weight { 1_000_000_000 }
+	fn payout_stakers_alive_controller(_n: u32, ) -> Weight { 1_000_000_000 }
+	fn rebond(_l: u32, ) -> Weight { 1_000_000_000 }
+	fn set_history_depth(_e: u32, ) -> Weight { 1_000_000_000 }
+	fn reap_stash(_s: u32, ) -> Weight { 1_000_000_000 }
+	fn new_era(_v: u32, _n: u32, ) -> Weight { 1_000_000_000 }
+	fn do_slash() -> Weight { 1_000_000_000 }
+	fn payout_all(_v: u32, _n: u32, ) -> Weight { 1_000_000_000 }
+	fn submit_solution_initial(_v: u32, _n: u32, _a: u32, _w: u32, ) -> Weight { 1_000_000_000 }
+	fn submit_solution_better(_v: u32, _n: u32, _a: u32, _w: u32, ) -> Weight { 1_000_000_000 }
+	fn submit_solution_weaker(_v: u32, _n: u32, ) -> Weight { 1_000_000_000 }
+}
```

Comment:
```suggestion
```

Reply:
And then we can remove this.

Reply:
Let's resolve this in the comment above.